.github/workflows/macos.yml

```diff
@@ -5,11 +5,17 @@ on: [push, pull_request]
 jobs:
   build:
     runs-on: macos-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby: [ 2.7, 2.4 ]
```

Comment:
Can you add 2.6 and 2.5 to this.

Reply:
I can if you'd like.  Previously, just one job was ran, and it was the default Actions Ruby.

MacOS jobs have a much lower concurrent limit that Ubuntu or Windows jobs, so I thought I'd run low/high...

Reply:
Done